modules/tinymce/src/plugins/lists/test/ts/browser/ApplyToDivTest.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable mocha/no-exclusive-tests */
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinySelections, TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/lists/Plugin';
+
+describe('browser.tinymce.plugins.lists.ApplyToDivTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'lists',
+    forced_root_block: 'div',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin ]);
+
+  it('TINY-9872: Apply to divs in forced_root_block:div', () => {
+    const editor = hook.editor();
+    editor.setContent('<div>Line 1</div><div>Line 2</div><div>Line 3</div>');
+
+    TinySelections.setSelection(editor, [ 0, 0 ], 2, [ 2, 0 ], 2);
+    editor.execCommand('InsertUnorderedList');
+
+    TinyAssertions.assertContent(editor, '<ul>\n<li>Line 1</li>\n<li>Line 2</li>\n<li>Line 3</li>\n</ul>');
+    TinyAssertions.assertSelection(editor, [ 0, 0, 0 ], 2, [ 0, 2, 0 ], 2);
+  });
+
+  it('TINY-9872: Apply to divs in forced_root_block:div, but not when widely separated', () => {
```

Comment:
The `forced_root_block` option doesn't seem to have an affect to this issue? Should you specifically check for that?

Reply:
Well, true, the forced div isn't actually the cause of the issue, that is a poorly written name on my part. My bad.